api/app/jobs/update_hackbot_slack_username_job.rb

```
@@ -4,7 +4,11 @@ class UpdateHackbotSlackUsernameJob < ApplicationJob
   def perform(*)
     Hackbot::Team.find_each do |team|
       info = ::SlackClient::Users.info(team.bot_user_id, team.bot_access_token)
-      team.update(bot_username: info[:user][:name])
+
+      # https://api.slack.com/methods/users.info
+      unless info[:error] == 'account_inactive'
+        team.update(bot_username: info[:user][:name])
```

What was the justification for failing silently and not logging here?